internal/cloud/openssh/sshshim/logger.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/natefinch/lumberjack"
```

Comment:
This should be `gopkg.in/natefinch/lumberjack.v2`. no?

(from readme in https://github.com/natefinch/lumberjack)